data.js

```diff
@@ -776,21 +793,78 @@ const municipiosObj = {
   ],
   "Zacapa": [
     "Zacapa",
-    "Cabañas",
     "Estanzuela",
-    "Gualán",
-    "Huité",
-    "La Unión",
     "Río Hondo",
-    "San Diego",
-    "San Jorge",
+    "Gualán",
     "Teculután",
     "Usumatlán",
+    "Cabañas",
+    "San Diego",
+    "La Unión",
+    "Huité",
+    "San Jorge",
   ],
 }
+/** # Población por departamento
+ * Devuelve como resultado un entero con la catidad total de la población por departamento, sin diacríticos.
+ */
+const poblacionDepto = {
+  "altaverapaz": "1,215,038",
+  "bajaverapaz": "299,476",
+  "chimaltenango": "615,776",
+  "chiquimula": "415,063",
+  "elprogreso": "176,632",
+  "escuintla": "733,181",
+  "guatemala": "3,015,081",
+  "huehuetenango": "1,170,669",
+  "izabal": "408,688",
+  "jalapa": "342,923",
+  "jutiapa": "488,395",
+  "peten": "545,600",
+  "quetzaltenango": "799,101",
+  "quiche": "949,261",
+  "retalhuleu": "326,828",
+  "sacatepequez": "330,469",
+  "sanmarcos": "1,032,277",
+  "santarosa": "396,607",
+  "solola": "421,583",
+  "suchitepequez": "554,695",
+  "totonicapan": "418,569",
+  "zacapa": "245,374",
+}
+
+/** # Población por departamento
+ *  Devuelve como resultado un entero con la catidad total de la población por departamento
+ */
+const poblacionDeptoObj = {
+  "Alta Verapaz": "1,215,038",
+  "Baja Verapaz": "299,476",
+  "Chimaltenango": "615,776",
+  "Chiquimula": "415,063",
+  "El Progreso": "176,632",
+  "Escuintla": "733,181",
+  "Guatemala": "3,015,081",
+  "Huehuetenango": "1,170,669",
+  "Izabal": "408,688",
+  "Jalapa": "342,923",
+  "Jutiapa": "488,395",
+  "Petén": "545,600",
+  "Quetzaltenango": "799,101",
+  "Quiché": "949,261",
+  "Retalhuleu": "326,828",
+  "Sacatepéquez": "330,469",
+  "San Marcos": "1,032,277",
+  "Santa Rosa": "396,607",
+  "Sololá": "421,583",
+  "Suchitepéquez": "554,695",
+  "Totonicapán": "418,569",
+  "Zacapa": "245,374",
+}
```

Comment:
Convert the numbers to "Number" type
It is more consistent for programming
User underscore (_) to separate the thousands if you want.
``` js
// Example
{
  "Sacatepéquez": 330469,
  "San Marcos": 1_032_277,
}
  
 ```

Reply:
Always as string data?

Reply:
Done